RELEASE_NOTES.md

```diff
@@ -481,19 +536,12 @@ include the new sigmoid module.  Options are now as follows:
   'fixed-size-text' template it is now possible to insert text with
   constant font size.
 
-- In the drawn mask blending mode there used to be an "invert mask"
-  option that had the same functionality as the "toggle polarity" option.
-  Since both were doing the same thing, "invert mask" is now removed.
-
 - Successive changes to sliders and other widgets (for example by
```

Comment:
This could be moved to the "Performance Improvements" section, as that was the main intent and result.

Reply:
moved